manager/assets/modext/widgets/element/modx.tree.element.js

```diff
@@ -563,6 +563,32 @@ Ext.extend(MODx.tree.Element,MODx.tree.Tree,{
             });
         }
 
+        if (n.isLoaded()) {
+            var childNodes = n.childNodes;
+
+            if (childNodes.some(function(child) { return !child.leaf; })) { // If any childNode has own children
```

Comment:
If you use cool array functions, why not to use arrow functions?
`if (childNodes.some(function(child) { return !child.leaf; })) {`
VVV
`if (childNodes.some(child => !child.leaf)) {`

Reply:
You came right after ES6 courses?! Do you see any polyfill in MODX?